### DISCUSSION TITLE
How to toggle the Browser Manager Sidebar on button press?

### DISCUSSION THREAD

**Discussion by Test-Account666** [Floorp-Projects/Floorp#1054] [Q&A]
I like the browser manager sidebar, but it is also kinda annoying when I don't need it.

I'd like to have a way to toggle it. I know there's custom actions, but I don't know how to set these up.

This is the sidebar I'm talking about btw:
![Sidebar](https://github.com/Floorp-Projects/Floorp/assets/36412486/c268bef0-81cd-4ac3-a6a5-96ef49978232)

**Comment by breadbeforerice** [comment]
https://github.com/floorp-projects/floorp/discussions/556 #discussionComment-7465366 Used based on this. However, if this setting alone is opened and closed, the lower part is sloppy and on the way.
![2024_3_31_10_28_7_11698](https://github.com/Floorp-Projects/Floorp/assets/158130154/3793a76a-b730-4df1-b79d-4d4be7f83845)
